Sources/MapboxNavigation/BannerContainerView.swift

```diff
@@ -4,14 +4,16 @@ import UIKit
 @objc(MBBannerContainerView)
 open class BannerContainerView: UIView {
     
+    // :nodoc:
     public enum `Type` {
```

Comment:
These APIs are for internal consumption for now (even if they're using on application level), so not documenting them.